vector/src/main/scala/geotrellis/vector/affine/package.scala

```diff
@@ -0,0 +1,56 @@
+package geotrellis.vector
+
+import com.vividsolutions.jts.{geom => jts}
+
+package object affine {
+
+  implicit class AffineTransformationPolygon(geom: Polygon) {
+    import com.vividsolutions.jts.geom.util.AffineTransformation._
```

Comment:
What's the reasoning for putting the import inside of the class, instead of at the top?